addons/mail/static/src/core/common/message_model.js

```diff
@@ -52,10 +52,7 @@ export class Message extends Record {
                 return false;
             }
             const otherDidNotSee = this.thread.channelMembers.filter((member) => {
-                return (
-                    member.persona.notEq(this.author) &&
-                    (!member.seen_message_id || member.seen_message_id.id < this.id)
-                );
+                return member.hasSeen(this);
```

Comment:
Could remove curly braces and the return statement, it would be easier to read :)

Reply:
Also, we don't need to filter, using some would be more efficient/readable:

```js
return members.some((m) => !m.hasSeen(this))
```